<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>550</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Worker</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <selectedon>:/img/src/image/icon_1.png</selectedon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(179, 200, 207);</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>451</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QComboBox" name="position_comboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>241</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caros Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
                background-color: rgb(255, 255, 255);
                border: 2px solid  rgb(0, 31, 63);
                border-radius: 8px;
                padding: 5px;
                font-size: 14pt;
                color: rgb(50, 50, 50);
            }
            QComboBox::drop-down {
                background-color: rgb(0, 31, 63);
                border: none;
                width: 25px;
            }
            QComboBox::down-arrow {
                image: url(:/img/src/image/chevron_down.png); 
                width: 12px;
                height: 12px;
            }</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="hire_date_dateEdit">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>300</y>
      <width>241</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caros Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateEdit {
                background-color: rgb(255, 255, 255);
                border: 2px solid  rgb(0, 31, 63);
                border-radius: 8px;
                padding: 5px;
                font-size: 14pt;
                color: rgb(50, 50, 50);
            }
QDateEdit::drop-down {
                background-color: rgb(0, 31, 63);
                border: none;
                width: 25px;
            }
            QDateEdit::down-arrow {
                image: url(:/img/src/image/chevron_down.png); 
                width: 12px;
                height: 12px;
            }
            QDateEdit QCalendarWidget {
                background-color: rgb(240, 240, 240);
                border: 1px solid rgb(200, 200, 200);
                font-size: 12pt;
                color: rgb(0, 0, 0);
				font: 25 16pt &quot;Caros Light&quot;;
            }
            QDateEdit QCalendarWidget QToolButton {
                color: rgb(0, 31, 63);
                font-size: 12pt;
                border: none;
                margin: 5px;
                padding: 5px;
                border-radius: 5px;
				font: 25 16pt &quot;Caros Light&quot;;
            }
            QDateEdit QCalendarWidget QToolButton:hover {
                background-color: rgb(58, 109, 140);
				font: 25 16pt &quot;Caros Light&quot;;
            }
            QDateEdit QCalendarWidget QTableView {
                background-color: white;
                color: black;
				font: 25 16pt &quot;Caros Light&quot;;
            }
QDateEdit QCalendarWidget QTableView:selected {
                background-color: rgb(0, 122, 204);
                color: white;
				font: 25 16pt &quot;Caros Light&quot;;
 }
QDateEdit QCalendarWidget QToolButton#qt_calendar_prevmonth  {
	qproperty-icon: none;
    image: url(:/img/src/image/Left.png);
}

QDateEdit QCalendarWidget QToolButton#qt_calendar_nextmonth {
	qproperty-icon: none;
    image: url(:/img/src/image/Right.png); 
}
</string>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>1945</year>
      <month>9</month>
      <day>14</day>
     </datetime>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caros Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(0, 31, 63);</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>60</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caros Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(0, 31, 63);</string>
    </property>
    <property name="text">
     <string>PASSWORD</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>110</y>
      <width>88</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caros Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(0, 31, 63);</string>
    </property>
    <property name="text">
     <string>FULL NAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>210</y>
      <width>103</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caros Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(0, 31, 63);</string>
    </property>
    <property name="text">
     <string>DEPARTMENT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>260</y>
      <width>78</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caros Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(0, 31, 63);</string>
    </property>
    <property name="text">
     <string>POSITION</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>310</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caros Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(0, 31, 63);</string>
    </property>
    <property name="text">
     <string>HIRE DATE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>340</y>
      <width>96</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caros Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(0, 31, 63);</string>
    </property>
    <property name="text">
     <string>PREVIOUS EXPERIENCE</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>410</y>
      <width>91</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caros Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(0, 31, 63);</string>
    </property>
    <property name="text">
     <string>BIRTH DATE</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="birth_date_dateEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>400</y>
      <width>241</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caros Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDateEdit {
                background-color: rgb(255, 255, 255);
                border: 2px solid  rgb(0, 31, 63);
                border-radius: 8px;
                padding: 5px;
                font-size: 14pt;
                color: rgb(50, 50, 50);
            }
QDateEdit::drop-down {
                background-color: rgb(0, 31, 63);
                border: none;
                width: 25px;
            }
            QDateEdit::down-arrow {
                image: url(:/img/src/image/chevron_down.png); 
                width: 12px;
                height: 12px;
            }
            QDateEdit QCalendarWidget {
                background-color: rgb(240, 240, 240);
                border: 1px solid rgb(200, 200, 200);
                font-size: 12pt;
                color: rgb(0, 0, 0);
				font: 25 16pt &quot;Caros Light&quot;;
            }
            QDateEdit QCalendarWidget QToolButton {
                color: rgb(0, 31, 63);
                font-size: 12pt;
                border: none;
                margin: 5px;
                padding: 5px;
                border-radius: 5px;
				font: 25 16pt &quot;Caros Light&quot;;
            }
            QDateEdit QCalendarWidget QToolButton:hover {
                background-color: rgb(58, 109, 140);
				font: 25 16pt &quot;Caros Light&quot;;
            }
            QDateEdit QCalendarWidget QTableView {
                background-color: white;
                color: black;
				font: 25 16pt &quot;Caros Light&quot;;
            }
QDateEdit QCalendarWidget QTableView:selected {
                background-color: rgb(0, 122, 204);
                color: white;
				font: 25 16pt &quot;Caros Light&quot;;
 }
QDateEdit QCalendarWidget QToolButton#qt_calendar_prevmonth  {
	qproperty-icon: none;
    image: url(:/img/src/image/Left.png);
}

QDateEdit QCalendarWidget QToolButton#qt_calendar_nextmonth {
	qproperty-icon: none;
    image: url(:/img/src/image/Right.png); 
}
</string>
    </property>
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="minimumDateTime">
     <datetime>
      <hour>0</hour>
      <minute>0</minute>
      <second>0</second>
      <year>1945</year>
      <month>9</month>
      <day>14</day>
     </datetime>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="experience_lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>350</y>
      <width>241</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(255, 255, 255);
    border: 2px solid  rgb(0, 31, 63);
    border-radius: 8px; 
    padding: 5px; 
	font-size: 10pt;
}</string>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="full_name_lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>241</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(255, 255, 255);
    border: 2px solid  rgb(0, 31, 63);
    border-radius: 8px; 
    padding: 5px; 
	font-size: 10pt;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>241</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(255, 255, 255);
    border: 2px solid  rgb(0, 31, 63);
    border-radius: 8px; 
    padding: 5px; 
	font-size: 10pt;
}</string>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="login_lineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>241</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caros Light</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	background-color: rgb(255, 255, 255);
    border: 2px solid  rgb(0, 31, 63);
    border-radius: 8px; 
    padding: 5px; 
	font-size: 10pt;
}</string>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QComboBox" name="department_comboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>241</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caros Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
                background-color: rgb(255, 255, 255);
                border: 2px solid  rgb(0, 31, 63);
                border-radius: 8px;
                padding: 5px;
                font-size: 14pt;
                color: rgb(50, 50, 50);
            }
            QComboBox::drop-down {
                background-color: rgb(0, 31, 63);
                border: none;
                width: 25px;
            }
            QComboBox::down-arrow {
                image: url(:/img/src/image/chevron_down.png); 
                width: 12px;
                height: 12px;
            }</string>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="sex_comboBox">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>241</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caros Light</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
                background-color: rgb(255, 255, 255);
                border: 2px solid  rgb(0, 31, 63);
                border-radius: 8px;
                padding: 5px;
                font-size: 14pt;
                color: rgb(50, 50, 50);
            }
            QComboBox::drop-down {
                background-color: rgb(0, 31, 63);
                border: none;
                width: 25px;
            }
            QComboBox::down-arrow {
                image: url(:/img/src/image/chevron_down.png); 
                width: 12px;
                height: 12px;
            }</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Caros Black</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:  rgb(0, 31, 63);</string>
    </property>
    <property name="text">
     <string>SEX</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="save_pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>480</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Caros Black</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color: rgb(0, 31, 63);
border-radius: 20px;
color: rgb(255, 255, 255);
}

QPushButton:hover{
background-color: rgb(58, 109, 140);
}</string>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="rec.qrc"/>
 </resources>
 <connections/>
</ui>
